actionkit/api/orderrecurring.py

```diff
@@ -0,0 +1,104 @@
+from actionkit.api.base import ActionKitAPI
+
+try:
+    import urllib.parse
+    encode_url = urllib.parse.urlencode
+except ImportError:
```

Comment:
Suggestion: make this Python 3-only, since Python 2 is no longer getting security updates.